pkg/api/vlabs/validate.go

```diff
@@ -699,7 +699,7 @@ func (a *Properties) validateNetworkPolicy() error {
 	}
 
 	// Temporary safety check, to be removed when Windows support is added.
-	if (networkPolicy == "calico" || networkPolicy == "azure") && a.HasWindows() {
+	if (networkPolicy == "calico") && a.HasWindows() {
```

Comment:
Nit: no need to have parenthesis here.

Reply:
done.

Reply:
Is parenthesis around (networkPolicy == "calico") removed?